src/pyprojroot/root.py

```diff
@@ -51,15 +55,18 @@ def find_root_with_reason(
     raise RuntimeError("Project root not found.")
 
 
-def find_root(criterion, start: _PathLike = None, **kwargs) -> _pathlib.Path:
+def find_root(
+    criterion: _CriterionType,
+    start: _typing.Union[None, _PathType] = None,
+) -> _pathlib.Path:
     """
     Find directory matching root criterion.
 
     Recursively search parents of start path for directory
     matching root criterion.
     """
     try:
-        root, _ = find_root_with_reason(criterion, start=start, **kwargs)
+        root, _ = find_root_with_reason(criterion, start=start)
```

Comment:
I'd prefer it if the extra `kwargs` we still passed through, since it ensures future compatibility. What was the mypy error?